app/src/main/res/layout/activity_main.xml

```diff
@@ -214,5 +214,11 @@
 
         </LinearLayout>
 
+        <Button
```

Comment:
I expected to see the settings button in the header of the application (after the app name).
Please move it there if it's easy and fast to do.